appium/webdriver/common/multi_action.py

```diff
@@ -45,7 +45,7 @@ def __init__(self, driver: 'WebDriver', element: Optional['WebElement'] = None)
         self._element = element
         self._touch_actions: List['TouchAction'] = []
 
-    def add(self, *touch_actions: 'TouchAction') -> None:
+    def add(self, *touch_actions: 'TouchAction') -> 'MultiAction':
```

Comment:
where does it return?

Reply:
Updated. Just docstring return value updated now to align with the return value (None). Please check.

Reply:
we plan anyway to [remove](https://github.com/appium/python-client/pull/960) these endpoints soon, but for the sake of correctness lets have this patch merged

Reply:
Yes, I'm aware of that. After seeing it in the issues, I thought why not correct it for now.